docs/TeamCommands.md

```diff
@@ -84,19 +83,28 @@ easy slack autocomplete.
 /rocket team remove GITHUB_TEAM_NAME SLACK_ID
 ```
 
-> **Note:** This command does not remove a member of the team from Github.
-
-Remove a user from a team. Team Leads can only remove users from teams that they
+Remove a user from a team, removes them as Team Lead if they were one.
+Team Leads can only remove users from teams that they
 are a part of, but admins can remove users from any team. `SLACK_ID` is the
 `@`-name, for easy slack autocomplete.
 
+### Lead (Team Lead\* and Admin only)
+
+```sh
+/rocket team lead GITHUB_TEAM_NAME SLACK_ID [--remove]
```

Comment:
Perhaps there should be an additional command to check the lead of a given team? I suppose that's listed in `view` though.

Reply:
Yah I guess it would be shown in view